src/components/AgentAuthority/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js

```diff
@@ -1,22 +1,99 @@
 /**
- * This function is responsible for rendering the left content column of the screen.
- * @returns The `ColumnLeftWrap` component returns a `div` element with a `div`
- * element inside of it.
+ * This function is used to display issued credential records log
  */
+import { useEffect, useState } from 'react'
+import IssuedRecordItem from '../IssuedRecordItem'
+import useGetIssueCredentialRecords from '../../../../interface/hooks/use-get-looped-issue-credential-records'
+
+export default function ColumnLeftWrap({ origin }) {
+  const [dataRecords, setDataRecords] = useState([])
+  const [flyerIdSearch, setFlyerIdSearch] = useState('')
+  const [statusRecords, errorRecords, startGetRecordsHandler] =
+    useGetIssueCredentialRecords()
+  useEffect(() => {
+    const setStoreDataFn = (resData) => setDataRecords(resData)
+    const intervalIdFetch = startGetRecordsHandler(origin, setStoreDataFn)
+    if (statusRecords !== 'started') clearInterval(intervalIdFetch)
+    return function clear() {
+      return clearInterval(intervalIdFetch)
+    }
+  }, [origin, statusRecords, startGetRecordsHandler])
+
+  const inputNameHandler = (e) => {
+    setFlyerIdSearch(e.target.value)
+  }
+
+  const renderedRecords = dataRecords.filter((record) => {
+    const flyerId = record.cred_ex_record.cred_preview.attributes[0].value
+    return flyerId.indexOf(flyerIdSearch) !== -1
+  })
 
-export default function ColumnLeftWrap() {
   return (
     <>
       <div className="col-md-6">
         <div className="container py-1">
           <div className="row">
             <div className="col-md-12">
-              <h5>Enter Flyer ID</h5>
-              <p className="small">View all certificates for a flyer ID</p>
+              <h5>Issued Licenses</h5>
+              <p className="small">Search for issued licenses by flyer ID</p>
+              <input
+                type="text"
+                value={flyerIdSearch}
+                onChange={inputNameHandler}
+                required
+                className="form-control"
+                id="name"
+                placeholder="GBR-RP-123456"
+              />
+              <div className="container py-3">
+                <div className="accordion" id="accordion">
+                  {renderedRecords.length === 0 && (
+                    <p className="small">No credentials found</p>
+                  )}
+
+                  {renderedRecords.length > 0 && (
+                    <>
```

Comment:
does it need to be wrapped in `<Framgment>`?

Reply:
Also it repeats, maybe we could do function?


```js
renderCard (renderRecords) => renderRecords.length > 0 ? <div className='card m0 p0'></div> : null;
```

Reply:
`<>` is shorthand for a fragment

Reply:
and the repeat was already there I believe